### ISSUE TITLE
[WebUI] Need increase width of fields in Status page 

### ISSUE THREAD

**Issue by dehnli** [vstakhov/rspamd#1186]
If amount of messages more than 100k `k` symbol doesn't fit
```
# rspamd --version
Rspamd daemon version 1.4.1-git.6d4ae4e
```
```
# rspamc stat                                                
Results for command: stat (0.086 seconds)
Messages scanned: 791464
...
Messages with action greylist: 10993, 1.38%
Messages with action no action: 591356, 74.71%
...
```
 
![rspamd-webui-status-section](https://cloud.githubusercontent.com/assets/14199981/20641481/1a105cde-b412-11e6-8805-018ba74bd2cf.png)

**Comment by dehnli** [comment]
This was after [#6d4ae4e](https://github.com/vstakhov/rspamd/commit/6d4ae4e88e53223b4b2ae646d5cf3fa4703f42e0)

**Comment by dehnli** [comment]
Fixed by [5ac8aec](https://github.com/vstakhov/rspamd/commit/5ac8aecac7e087c2f9abe4161f01b2b1b94416d2)
